controller_manager/src/controller_manager.cpp

```diff
@@ -29,6 +31,11 @@
 namespace controller_manager
 {
 
+bool isControllerRunning(controller_interface::ControllerInterface & controller)
```

Comment:
Yeah, there's a bit of mixup between the old nomenclature and the new one.

A controller can be unloaded or loaded.
And once loaded it can be configured, active, inactive and unconfigured.

Running corresponds to loaded + active, and means it is called in the update loop.

I need to rename this function since it doesn't respect the naming standard anyway, I can change it to `is_controller_active`.